### PR TITLE
Fix running CML inside CML with `pkg` binaries

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Fixes https://github.com/iterative/cml/issues/920, but is just a hack; don't merge unless we don't find a better solution.

### CODE REVIEW

File: iterative/utils/helpers.go [hunk 1]
```diff
@@ -44,7 +44,8 @@ func getGHCML(v string) string {
 	return fmt.Sprintf(`sudo mkdir -p /opt/cml/
 sudo curl --location --url %s --output /opt/cml/cml-linux
 sudo chmod +x /opt/cml/cml-linux
-sudo ln -s /opt/cml/cml-linux /usr/bin/cml`, v)
+sudo ln -s /opt/cml/cml-linux /usr/bin/cml
+sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v)
```
Comment (casperdcl) [inline iterative/utils/helpers.go:48]:
```suggestion
sudo ln -s /opt/cml/cml-linux /usr/bin/cml-internal`, v)
```

**Review by casperdcl — CHANGES_REQUESTED** [review]
> don't merge unless we don't find a better solution

this isn't not double negation backfiring

File: iterative/utils/helpers.go (hunk 1 above)
Comment (0x2b3bfa0) [reply iterative/utils/helpers.go:48]: Not this time. The whole point of this pull request is having two names for the same file.

Comment (0x2b3bfa0) [reply iterative/utils/helpers.go:48]: Soft links are ineffective because they get resolved and the problem is the same.

**Comment by 0x2b3bfa0** [comment]
I'm not denying that this pull request must not be left unmerged.

File: iterative/utils/helpers.go (hunk 1 above)
Comment (0x2b3bfa0) [reply iterative/utils/helpers.go:48]: 8f233c117e9ae088a8822dae26d1cfba61d756c2 — titled _“Add `cml-internal` hard link”_

Comment (casperdcl) [reply iterative/utils/helpers.go:48]: kewl beans

**Review by casperdcl — COMMENTED** [review]
okay but this level of hackery requires an inline comment methinks

File: iterative/utils/helpers.go (hunk 1 above)
Comment (0x2b3bfa0) [reply iterative/utils/helpers.go:48]:
```suggestion
sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v) # hard link to fix cml#920
```

File: iterative/utils/helpers.go [hunk 2]
```diff
@@ -44,7 +44,8 @@ func getGHCML(v string) string {
 	return fmt.Sprintf(`sudo mkdir -p /opt/cml/
 sudo curl --location --url %s --output /opt/cml/cml-linux
 sudo chmod +x /opt/cml/cml-linux
-sudo ln -s /opt/cml/cml-linux /usr/bin/cml`, v)
+sudo ln -s /opt/cml/cml-linux /usr/bin/cml
+sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v) # hard link to fix cml#920
```
Comment (0x2b3bfa0) [inline iterative/utils/helpers.go:48]:
```suggestion
sudo ln /opt/cml/cml-linux /usr/bin/cml-internal`, v) // hard link to fix cml#920
```

**Comment by dacbd** [comment]
The alternative route I was pursuing was (partially) unsuccessful, it might still be possible but it requires more tinkering.

**Review by dacbd — APPROVED** [review]

**Review by casperdcl — APPROVED** [review]
:)

**Pull request merged**